examples/java/pom.xml

```diff
@@ -194,6 +198,24 @@
           </execution>
         </executions>
       </plugin>
+
+      <!-- Override parent configuration.
+           Spark/Flink runner dependencies are necessary to be in shaded jar
+           in order to run WordCountIT with Spark/Flink runner -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>2.10</version>
```

Comment:
Yes, but actually I remove the `failOnWarning` setting to make it as default (`false`). `failOnWarning` makes the project build failed due to unused spark and flink runner dependency.